iron-resizable-behavior.html

```diff
@@ -202,6 +179,36 @@
       this._notifyingDescendant = true;
       descendant.notifyResize();
       this._notifyingDescendant = false;
+    },
+    
+    _requestResizeNotifications: function() {
+      if (!this.isAttached)
```

Comment:
I kept implementation parity compared to before. It could be that this element might be detached before the document `readystatechange` event